package.json

```diff
@@ -10,7 +10,7 @@
   "repository": "https://github.com/jschr/electron-react-redux-boilerplate",
   "license": "MIT",
   "dependencies": {
-    "babel-runtime": "^6.26.0",
```

Comment:
I didn't find any need for babel-runtime in modern Node.js. I may be wrong, but so far so good :)